### PR TITLE
Fix Spelling

### PR DESCRIPTION
**Pull request by Kaleidosium**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +58,14 @@ To build, simply run the following commands from the `Source` directory:
 	cmake ..
 	make
 
-The compiled binaries should be appear in the directory named `build`.
+The compiled binaries should appear in the directory named `build`.
 
 ## General usage
-Download the precalculation file provided along the binaries release for the game you intend to use the manipulation and extract it into the program's directory. Then, select the game in the dropdown list and start the seed finding procedure by clicking on "Find your seed". This will show a wizard to guide you through the process. If it succeeds, you will be able to see your starters prodictions depending on the amount of frames it takes to press A on the new game confirmation prompt and pressing A on the name confirmation prompt. You can set the desired predictions you wish to get by configuring the filters in the settings and you can also reroll the predictions as many times as you want providing you generate another random battle team.
```
Comment (aldelaro5) [inline README.md]: My English might be wrong, but this change doesn't seem to make sense to me.

What I want to say is that TO reroll, you NEED to generate an additional random battle team, the change here, I am not sure if it's correct syntax wise.

**Review by aldelaro5 — CHANGES_REQUESTED** [review]
Other than that, LGTM

File: README.md (hunk 1 above)
Comment (Kaleidosium) [reply README.md]: Reverted it to the older version.

**Comment by aldelaro5** [comment]
Please squash the commits

**Comment by Kaleidosium** [comment]
There we go. please merge, I had a headache doing this.

**Review by aldelaro5 — APPROVED** [review]

**Pull request merged**